### PR TITLE
Initial commit of sslh

### PR DESCRIPTION
**Pull request by benf33**
New component : sslh

### CODE REVIEW

File: components/network/sslh/Makefile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+COMPONENT_CLASSIFICATION= System/Security
+
+
+COMPONENT_LICENSE_FILE= COPYING
+COMPONENT_LICENSE=	GPLv2
+
+PATH=/usr/gnu/bin:/usr/bin:/usr/perl5/bin
+LD_OPTIONS=-lresolv -lsocket -lnsl
+
+include ../../../make-rules/prep.mk
```
Comment (pyhalov) [inline components/network/sslh/Makefile:37]: Use $(WS_TOP)

File: components/network/sslh/sslh.p5m [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license COPYING license=GPLv2
+
+file Solaris/sslh.xml path=lib/svc/manifest/network/sslh.xml
+file Solaris/sslh.sh path=lib/svc/method/sslh
+file path=usr/sbin/sslh
+file path=usr/share/man/man8/sslh.8
```
Comment (pyhalov) [inline components/network/sslh/sslh.p5m:27]: Can we provide some sample /etc/sslh.cfg  config?

File: components/network/sslh/Makefile (hunk 1 above)
Comment (benf33) [inline components/network/sslh/Makefile:37]: Should I update libconfig as well with another PR ?

File: components/network/sslh/sslh.p5m (hunk 2 above)
Comment (benf33) [inline components/network/sslh/sslh.p5m:27]: I was wondering in /etc or in /usr/share/doc ?

Comment (pyhalov) [inline components/network/sslh/sslh.p5m:27]: It depends. If service can reasonably work with this file, I'd keep it in /etc. If it needs customizing in most cases, I'd leave it in /usr/share/doc/sslh/.

File: components/network/sslh/Makefile (hunk 1 above)
Comment (pyhalov) [inline components/network/sslh/Makefile:37]: I think it's not necessary now.

**Comment by alarcher** [comment]
Any update ?

**Comment by benf33** [comment]
Sorry, I do not have time to work on OI right now. But it is still on my roadmap.

**Comment by xen0l** [comment]
@benf33 we will close this PR for now. Once, you will work on it again, feel free to open it.

**Pull request closed without merging**